ios/Video/RCTVideo.swift

```diff
@@ -1205,7 +1206,8 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
         return
     }
 
-    func removeReactSubview(subview: UIView!) {
+    override func removeReactSubview(_ subview: UIView!) {
+        super.removeReactSubview(subview)
```

Comment:
Same applies here for the remove

Reply:
I don't think this callback was ever triggered before. Looking into it